docs/instruments/instrumentserver.md

```diff
@@ -141,3 +146,82 @@ Changing things in the GUI will also be reflected in the code.
     Changing something from the GUI only changes the code if we are calling the parameter manager directly.
     If we store a value in a separate variable and then change the GUI, the value in the variable might not get update.
     Because of this, we always recommend to call the Parameter Manager directly instead of saving the values in variables.
+
+## Config Files
+
+The instrumentserver also contains functionality for config files. These config files contain set-up for QCoDeS instruments. These config files can then be used by the instrumentserver 
+on start-up to have the desired instruments already created without having to manually add them after starting the server.
+
+To open the instrument server with a config file, we can use the '-c' flag to specify a config file, followed by the path to the config file:
+
+```bash
+$ instrumentserver -c serverConfig.yml
+```
+
+
+Below is a sample config file to initialize a DummyInstrument and an Oxford Triton dilution refrigerator
+
+`instruments`: this field will contain each instrument you wish to create. Under this field should be the names of each instrument you are creating
+
+`type`: This specifies which QCoDeS driver the instrumentserver should use to create the instrument. Can either be a local file or a community-created driver
+
+`address`: the tcp/ip address of the instrument on the same network as the instrumentserver
```

Comment:
This is whatever the instrument uses as an address. Sometimes its a tcp/ip other times a usb adress others a serial port or even the serial number if that driver uses that